README.md

```diff
@@ -28,17 +28,12 @@ To install Akeneo PIM for a PIM project or for evaluation, please follow: https:
 
 Alternatively, you can install Akeneo PIM with Composer, but please make sure that all requirements are fulfilled.
 
-If you don't have Composer yet, download it following the instructions on http://getcomposer.org/ or just run the following command:
-
-```
-    $ curl -s https://getcomposer.org/installer | php
-```
 
 Please note that you will certainly need to provide your GitHub credentials with this method.
 A lot of our dependencies are coming from GitHub and this reaches the max limit of 50 API calls from anonymous users.
 
 ```
-    $ php composer.phar create-project --prefer-dist akeneo/pim-community-standard ./pim-project "3.1.*@stable"
+    $ php composer.phar create-project --prefer-dist akeneo/pim-community-standard ./pim-project "4.0.*@stable"
```

Comment:
in case of a migration, we should explain it will copy some files from akeneo/dev (maybe not like that)
and that they should analyze the diff (if any)